bedrock/firefox/templates/firefox/new/scene1.html

```diff
@@ -0,0 +1,144 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/base-resp.html" %}
```

Comment:
Could we make the scene1 and scene 2 templates more dry if they extended their own base template perhaps? Seems like a lot of the page meta stuff could probably be shared?